meraki_devices.tf

```
@@ -516,3 +516,36 @@ resource "meraki_device_cellular_sims" "devices_cellular_sims" {
   sim_failover_enabled = each.value.sim_failover_enabled
   sim_failover_timeout = each.value.sim_failover_timeout
 }
+
+locals {
+  devices_wireless_radio_settings = flatten([
+    for domain in try(local.meraki.domains, []) : [
+      for organization in try(domain.organizations, []) : [
+        for network in try(organization.networks, []) : [
+          for device in try(network.devices, []) : {
+            key                                = format("%s/%s/%s/%s", domain.name, organization.name, network.name, device.name)
+            serial                             = meraki_device.devices[format("%s/%s/%s/%s", domain.name, organization.name, network.name, device.name)].serial
+            rf_profile_id                      = try(meraki_wireless_rf_profile.networks_wireless_rf_profiles[format("%s/%s/%s/%s", domain.name, organization.name, network.name, try(device.wireless.radio_settings.rf_profile_name, local.defaults.meraki.domains.organizations.networks.devices.wireless.radio_settings.rf_profile_name))].id, null)
```

We don't use defaults for names.
```suggestion
            rf_profile_id                      = try(meraki_wireless_rf_profile.networks_wireless_rf_profiles[format("%s/%s/%s/%s", domain.name, organization.name, network.name, device.wireless.radio_settings.rf_profile_name)].id, null)
```